roles/confluent.variables/defaults/main.yml

```diff
@@ -58,6 +58,27 @@ common_role_completed: false
 
 proxy_env: {}
 
+### The method of installation. Valid values are "package" or "archive". If "archive" is selected then services will not be installed via the use of yum or apt, but will instead be installed via expanding the target .tar.gz file from the Confluent archive into the path defined by `archive_destination_path`. Configuration files are also kept in this directory structure instead of `/etc`. SystemD service units are copied from the ardhive for each target service and overrides are created pointing at the new paths. The "package" installation method is the default behavior that utilizes yum/apt.
+installation_method: "package"
+
+### The Confluent Platform version of the archive to download. Something like *5.5.0* or *5.5.1*.
```

Comment:
"Something like" sounds unofficial. I think elsewhere i use "Possible values: 5.5.0, 5.5.1, etc"

Reply:
Done and I changed all other instances of "something like"